### ISSUE TITLE
studio: import and upload content from popular format like pptx

### ISSUE THREAD

**Issue by peterpeterparker** [deckgo/deckdeckgo#382]
## Description

It might useful, specially for editor who already have many existing presentations, to be able to import and upload content from popular format like pptx

**Comment by grantlouisherman** [comment]
hey @peterpeterparker do you care if I took a look at this?

**Comment by peterpeterparker** [comment]
sure @grantlouisherman always happy to get some help!

note: the description of the  issue, specially the "development" part might  now be up-to-date and correct anymore. I think we should treat it as nothing more than a blank page with the goal to be able to "import from pptx". open to idea and solution

**Comment by grantlouisherman** [comment]
yeah I think it will probably require a discussion. So you would want to simply import a Powerpoint and convert that to a DeckDeckGo presentation?

**Comment by peterpeterparker** [comment]
yes that's my understanding too. the idea is not mine but was suggested by our beta testers or more recently by some users.  so yes, being able  to provide a powerpoint and convert it to a deckdeckgo presentations.

to  be honest, it sounds  tricky to me, so  really, really  happy to get any help on  this!

**Comment by grantlouisherman** [comment]
Yeah in the browser is going to be interesting but let me poke around and I’ll get back to you. 👍

**Comment by peterpeterparker** [comment]
Sounds like a plan, thx!

**Comment by grantlouisherman** [comment]
hey @peterpeterparker so I just wanted to give you an update. I have learned that PowerPoints are in the Microsoft Open XML format. This format does differ from XML format in the sense that they have their own tags for how slides relate to each other. I found a few resources that I am going to take a look at this week, one of which is the Microsoft API for Open XML. 

For your reference:

- http://www.ericwhite.com/blog/open-xml-and-javascript/

- https://github.com/liamsi/Open-XML-SDK-for-JavaScript

**Comment by peterpeterparker** [comment]
really cool @grantlouisherman thank you for the feedback! such a weird format, it already sounded tricky before, even more know 😉 looking forward to hear about your discovery!
